codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -164,6 +161,18 @@ public int unconsumedBytes(Http2Stream stream) {
         return state(stream).unconsumedBytes();
     }
 
+    @Override
+    public void flushWindowUpdates(final ChannelHandlerContext ctx) throws Http2Exception {
+        connectionState().writeWindowUpdateIfNeeded(ctx);
+        connection.forEachActiveStream(new Http2StreamVisitor() {
```

Comment:
Do we have any performance numbers for this?  For example how many active stream do we have to have before this shows an impact?  Or is this impact overshadowed by other gains in this PR?